models/batch.go

```diff
@@ -14,4 +14,5 @@ type Batch struct {
 	AccountTreeRoot   *common.Hash
 	PrevStateRoot     *common.Hash
 	SubmissionTime    *Timestamp
+	Committer         common.Address
```

Comment:
I wouldn't add `Committer` to `models.Batch`. Note that this field will be equal to the zero address in many places, because we don't store the Committer in `stored.Batch`. We don't need to keep information about `Committer` for now either. So instead of adding `Committer` field to `models.Batch` I would make `getBatchDetails` return some new type and use it to initialize `DecodedBatchBase` later.